bl32/tsp/aarch64/tsp_entrypoint.S

```diff
@@ -263,6 +263,17 @@ func tsp_cpu_on_entry
 	mov	x0, #0
 	bl	bl32_plat_enable_mmu
 
+	/* ---------------------------------------------
+	 * Enable pointer authentication
+	 * ---------------------------------------------
+	 */
+#if ENABLE_PAUTH
+	mrs	x0, sctlr_el1
+	orr	x0, x0, #SCTLR_EnIA_BIT
+	msr	sctlr_el1, x0
+	isb
```

Comment:
Need the key programming to done again on warmboot as the TSPD clears the secure context on power down.